src/java/picard/vcf/GenotypeConcordanceCounts.java

```diff
@@ -74,6 +85,76 @@ public void validateCountsAgainstScheme(final GenotypeConcordanceScheme scheme)
     }
 
     /**
+     * Genotype Concordance is the number of times the truth and call states match exactly / all truth and call combinations made
+     * If the GA4GH scheme is being used, any MISSING sites in truth OR call will not be included in the discordance calculations.
+     */
+    public double calculateGenotypeConcordance(final GenotypeConcordanceScheme scheme, final boolean missingSitesFlag) {
+        double numerator = 0.0;
+        double denominator = 0.0;
+
+        scheme.validateScheme();
+
+        final TruthState[] allTruthStates = TruthState.values();
+        final CallState[] allCallStates = CallState.values();
+
+        for (final TruthState truthState : allTruthStates) {
+            for (final CallState callState : allCallStates) {
+                if(!missingSitesFlag && isMissing(truthState, callState)) {
+                    continue;
+                }
+                else {
+                    final TruthAndCallStates truthAndCallStates = new TruthAndCallStates(truthState, callState);
+                    final int count = getCount(truthAndCallStates);
+                    if (truthState.getCode()==callState.getCode()) {
+                        //Because the TruthState and CallState enums are in the same order, the ordinals correspond to the same constants
```

Comment:
2 minor points:
1. This comment no longer applies (since you aren't using ordinals anymore).
2. You can pull "denominator += count;" outside the if statement (since it's applied in either case) and get rid of the else block.